carma-messenger-config/chevrolet_tahoe_2018/docker-compose.yml

```diff
@@ -20,8 +20,6 @@ services:
     container_name: messenger_ros2
     volumes_from:
       - container:carma-config:ro
-    environment:
-      - CYCLONEDDS_URI=/opt/carma/vehicle/calibration/telematics/cyclone_config.xml
```

Comment:
Is there an error log that occurs when we try to use this? If so, can we add it to the Jira ticket?

Reply:
Yes issue is logged here - https://usdot-carma.atlassian.net/browse/WFD-608.